pkg/component/networking/vpn/seedserver/seedserver.go

```diff
@@ -195,19 +182,9 @@ func (v *vpnSeedServer) Deploy(ctx context.Context) error {
 				fileNameEnvoyConfig: v.getEnvoyConfig(),
 			},
 		}
-
-		dhSecret = &corev1.Secret{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      secretNameDH,
```

Comment:
Nit: You could also drop the const in https://github.com/gardener/gardener/blob/be23065e1441a703eef7f2ed565c865ab9f43017/pkg/component/networking/vpn/seedserver/seedserver.go#L76.